mixer/test/client/env/setup.go

```diff
@@ -283,6 +283,26 @@ func (s *TestSetup) VerifyReport(tag string, result string) {
 	}
 }
 
+// VerifyReport2 verifies Report request data.
+func (s *TestSetup) VerifyReport2(tag string, result1, result2 string) {
```

Comment:
Could you make this name more descriptive? How is this different from VerifyReport()?

Reply:
Renamed to VerifyTwoReports. It checks two reports in both possible orderings.